interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -845,6 +845,9 @@ def test_nondimensional(self):
         self.assertNear(np.dot(g.standard_gibbs_RT, g.X) - Smix_R,
                         g.gibbs_mole / (R*g.T))
 
+class TestPengRobinsonPhase(utilities.CanteraTest):
+    def setup(self):
+        self.gas = ct.Solution('co2_PR_exampl.yaml','CO2-PR')
```

Comment:
This test doesn't actually exercise any of the interface. Was that intentional, just testing whether you can create the class?

Reply:
The intent was to create a python test, which we forgot to add. I have added this test now.